deploy/run.sh

```diff
@@ -1,20 +1,24 @@
 #!/bin/bash
 
-set -e
-
-KUBE_ARGS=""
+set -ex
 
+# Check if InfluxDB service is running
 if [ ! -z $KUBERNETES_RO_SERVICE_HOST ]; then
-    echo "Detected Kube specific args. Starting in Kube mode."
-    KUBE_ARGS="--kubernetes_master $KUBERNETES_RO_SERVICE_HOST:$KUBERNETES_RO_SERVICE_PORT"
-fi
+  # TODO(vishh): add support for passing in user name and password.    
+  INFLUXDB_ADDRESS=""
+  if [ ! -z $MONITORING_INFLUXDB_SERVICE_HOST ]; then
+    INFLUXDB_ADDRESS="${MONITORING_INFLUXDB_SERVICE_HOST}:${MONITORING_INFLUXDB_SERVICE_PORT}"
```

Comment:
These variables are populated by the kubelet as part of static service
discovery. We can add more additional checks for sure, but it might not be
necessary.

On Wed, Jan 21, 2015 at 2:35 PM, Rohit Jnagal <notifications@github.com>
wrote:

> In deploy/run.sh
> <https://github.com/GoogleCloudPlatform/heapster/pull/66#discussion_r23340861>
> :
>
> >  if [ ! -z $KUBERNETES_RO_SERVICE_HOST ]; then
> > -    echo "Detected Kube specific args. Starting in Kube mode."
> > -    KUBE_ARGS="--kubernetes_master $KUBERNETES_RO_SERVICE_HOST:$KUBERNETES_RO_SERVICE_PORT"
> > -fi
> > +  # TODO(vishh): add support for passing in user name and password.
> > +  INFLUXDB_ADDRESS=""
> > +  if [ ! -z $MONITORING_INFLUXDB_SERVICE_HOST ]; then
> > +    INFLUXDB_ADDRESS="${MONITORING_INFLUXDB_SERVICE_HOST}:${MONITORING_INFLUXDB_SERVICE_PORT}"
>
> Do we need to check if a port was passed in
> (MONITORING_INFLUXDB_SERVICE_PORT)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/heapster/pull/66/files#r23340861>.
>